src/EnergyPlus/DElightManagerF.cc

```diff
@@ -262,460 +244,354 @@ namespace DElightManagerF {
-								ShowSevereError( "...Z Reference Point= " + RoundSigDigits( RefPt_WCS_Coord( 3 ), 2 ) + ", Zone Minimum Z= " + RoundSigDigits( Zone( izone ).MinimumZ, 2 ) + ", Zone Maximum Z= " + RoundSigDigits( Zone( izone ).MaximumZ, 2 ) );
-								ErrorsFound = true;
-							}
-
-							// Write this RefPt to the DElight input file
-
-							// Remove any blanks from the RefPt Name for ease of input to DElight
-							cNameWOBlanks = ReplaceBlanksWithUnderscores( AlphaArrayRefPt( 1 ) );
-							iLtgCtrlType = RealNumArrayDElight( 1 );
-							//                            write(unit,913) iNumRefPts, &
-							gio::write( unit, Format_913 ) << cNameWOBlanks << RefPt_WCS_Coord( 1 ) * M2FT << RefPt_WCS_Coord( 2 ) * M2FT << RefPt_WCS_Coord( 3 ) * M2FT << RealNumArrayRefPt( 4 ) << RealNumArrayRefPt( 5 ) * LUX2FC << iLtgCtrlType;
-
-							//                                        'Reference_Point ',I4,/, &
 
-							// RJH 2008-03-07: Set up DaylIllumAtRefPt for output for this DElight zone RefPt
+								// Write this RefPt to the DElight input file
 
-							// CurrentModuleObject='Daylighting:DELight:ReferencePoint'
-							SetupOutputVariable( "Daylighting Reference Point Illuminance [lux]", ZoneDaylight( izone ).DaylIllumAtRefPt( iNumRefPts ), "Zone", "Average", AlphaArrayRefPt( 1 ) );
-
-							// Predefined Reporting For Lighting Summary Report
-							PreDefTableEntry( pdchDyLtZone, AlphaArrayRefPt( 1 ), AlphaArrayDElight( 2 ) );
-							PreDefTableEntry( pdchDyLtKind, AlphaArrayRefPt( 1 ), "DElight" );
-							// (1=continuous, 2=stepped, 3=continuous/off)
-							{ auto const SELECT_CASE_var( int( RealNumArrayDElight( 1 ) ) );
-							if ( SELECT_CASE_var == 1 ) {
-								PreDefTableEntry( pdchDyLtCtrl, AlphaArrayRefPt( 1 ), "Continuous" );
-							} else if ( SELECT_CASE_var == 2 ) {
-								PreDefTableEntry( pdchDyLtCtrl, AlphaArrayRefPt( 1 ), "Stepped" );
-							} else if ( SELECT_CASE_var == 3 ) {
-								PreDefTableEntry( pdchDyLtCtrl, AlphaArrayRefPt( 1 ), "Continuous/Off" );
-							}}
-							PreDefTableEntry( pdchDyLtFrac, AlphaArrayRefPt( 1 ), RealNumArrayRefPt( 4 ) );
-							PreDefTableEntry( pdchDyLtWInst, AlphaArrayRefPt( 1 ), rLightLevel );
-							PreDefTableEntry( pdchDyLtWCtrl, AlphaArrayRefPt( 1 ), rLightLevel * RealNumArrayRefPt( 4 ) );
-
-						} // Max 100 RefPt test
-
-					} // RefPt in current DElight Zone test
-
-				} // Ref Pt loop
+								// Remove any blanks from the RefPt Name for ease of input to DElight
+								cNameWOBlanks = ReplaceBlanksWithUnderscores( refPt.Name );
+								if ( refPt.indexToFracAndIllum != 0 ){
+									gio::write( unit, Format_913 ) << cNameWOBlanks << RefPt_WCS_Coord( 1 ) * M2FT << RefPt_WCS_Coord( 2 ) * M2FT << RefPt_WCS_Coord( 3 ) * M2FT << znDayl.FracZoneDaylit( refPt.indexToFracAndIllum ) << znDayl.IllumSetPoint( refPt.indexToFracAndIllum ) * LUX2FC << znDayl.LightControlType;
```

Comment:
@Myoldmopar The line to write out the reference points is inserted here.  What's strange is this code runs fine on windows and there is no diff between the delight in file from this code vs v8.5 except for the number of control steps which is a separate issue.